src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -451,7 +470,12 @@ public function getSName()
      */
     public function getName()
     {
-        return $this->fname.' '.$this->sname;
+        if (empty($this->nname)) {
+            return $this->fname.' '.$this->sname;
+        } else {
+            return $this->fname.' "'.$this->nname.'" '.$this->sname;
+        }
+        return $this->fname.' '.$this->nname.' '.$this->sname;
```

Comment:
Minor note: you've got an `if { return } else { return }`, which will always return, then another `return` which will never get used.